examples/zephyr/fw_update/sample.yaml

```diff
@@ -10,10 +10,8 @@ tests:
     timeout: 600
     extra_args: EXTRA_CONF_FILE="../common/runtime_settings.conf"
     platform_allow: >
-      esp32_devkitc_wrover
```

Comment:
Is this change intentional?

Reply:
it was intentional to move this board to the build only. But as you've rightly identified, that's outside the scope of this PR so I'll remove this change before merging. Thanks!